doc/nrf/libraries/bluetooth_services/mesh/lightness_srv.rst

```diff
@@ -103,6 +103,14 @@ This information is used to reestablish the correct Light level when the device
 If :kconfig:option:`CONFIG_BT_SETTINGS` is enabled, the Light Lightness Server stores all its states persistently using a configurable storage delay to stagger storing.
 See :kconfig:option:`CONFIG_BT_MESH_MODEL_SRV_STORE_TIMEOUT`.
 
+The Light Lightness Server can use the :ref:`emergency data storage (EMDS) <emds_readme>` together with persistent storage to:
+
+* Extend the flash memory life expectancy.
+* Reduce the use of resources by reducing the number of writes to flash memory.
+
+If option :kconfig:option:`CONFIG_EMDS` is enabled, the Light Lightness Server continues to store the default light and light range states to the flash memory through the settings library, but the last known non-zero Light level and whether the light is on or off are stored by using the :ref:`EMDS <emds_readme>` library.
```

Comment:
Should we add a note say that if previously, models were storing the last states using settings, then switching to EMDS (e.g. via DFU) will reset previously stored values?